app/assets/stylesheets/layout/work_packages/_table_embedded.sass

```diff
@@ -97,5 +97,11 @@ $table-timeline--compact-row-height: 28px
     .wp-table--context-menu-th
       width: 25px
 
+  // Ensure embedded timelines share 50% width
+  // with table side
+  .work-packages-tabletimeline--table-side.-timeline-visible
+    max-width: 50%
+    overflow-x: scroll
```

Comment:
Is there a reason to for `overflow-x: scroll`? I guess `auto` would be better to avoid unnecessary scroll bars to be displayed.

Reply:
`auto` should work as well. but the default has `overflow-x: visible` which prints over the timeline

Reply:
Ah okay... then let's keep `scroll` 👍